.vscode/tasks.json

```diff
@@ -20,5 +46,9 @@
                 "clear": false
             }
         },
-    ]
+        {
+            "label": "Build Smoke Tests",
+            "dependsOn": ["tslint-smoke-tests", "build-smoke-tests"]
```

Comment:
I'd change `build-smoke-tests` to `compile-smoke-tests` to avoid ambiguity with  `Build Smoke Tests`

Reply:
Sure, makes sense.